### PR TITLE
Feature/bottom nav

### PR DESCRIPTION
**Pull request by memuyskens**
Looks like my commit history was goofed, but file changes are correct and we can squash the history when merging.

### CODE REVIEW

File: app/src/main/res/layout/fragment_notification.xml [hunk 1]
```diff
@@ -15,8 +15,8 @@
             android:id="@android:id/list"
             android:layout_width="match_parent"
             android:layout_height="match_parent"
-            android:dividerHeight="2dp"
-            android:divider="@color/accent"/>
+            android:dividerHeight="0dp"
```
Comment (ryanrampage1) [inline app/src/main/res/layout/fragment_notification.xml:18]: i would just delete the divider stuff

Comment (memuyskens) [reply app/src/main/res/layout/fragment_notification.xml:18]: I did try that first but I found out android automatically adds a thin, light-gray divider unless you explicitly make it 0dp w/ a drawable of @null.

Comment (memuyskens) [reply app/src/main/res/layout/fragment_notification.xml:18]: http://stackoverflow.com/questions/10163200/how-do-i-remove-the-divider-from-a-listview-on-android

Comment (ryanrampage1) [reply app/src/main/res/layout/fragment_notification.xml:18]: 👌🏼

**Review by ryanrampage1 — APPROVED** [review]
💯

**Pull request merged**